src/handlers/thirdparty.ts

```diff
@@ -49,6 +50,62 @@ export function handleThirdPartyAdded(event: ThirdPartyAdded): void {
   metric.save()
 }
 
+
+export function handleThirdPartyUpdated(event: ThirdPartyUpdated): void {
+  if (!isURNValid(event.params._thirdPartyId)) {
+    log.error('A third party was added with an invalid URN as an id "{}"', [
+      event.params._thirdPartyId
+    ])
+    return
+  }
+
+  let thirdParty = ThirdParty.load(event.params._thirdPartyId)
+
+  if (!thirdParty) {
+    log.error('Invalid third party "{}"', [
+      event.params._thirdPartyId
+    ])
+    return
+  }
+
+  thirdParty.resolver = event.params._resolver
+  thirdParty.rawMetadata = event.params._metadata
+
+  thirdParty.save()
+
+  let eventManagersAddresses = event.params._managers
```

Comment:
What if we have a specific method for the contract to update managers? Wouldn't it be cheaper to work with the contract taking into consideration that they would be the property that is most susceptible to change?